ui/dashboards/src/components/PanelDrawer/PanelEditorForm.tsx

```diff
@@ -72,7 +72,7 @@ export function PanelEditorForm(props: PanelEditorFormProps) {
 
   return (
     <form id={panelEditorFormId} onSubmit={handleSubmit}>
-      <Grid container spacing={2}>
+      <Grid container spacing={2} sx={{ overflowY: 'scroll', maxHeight: '90vh' }}>
```

Comment:
This is temporary, I added it so the user can scroll inside the edit panel drawer, can probably remove once I add separate tabs for 'Query' and 'Visualization' options

Reply:
Oh yeah I just added almost the same thing in a different PR (added it on the `form` element). We will need some kind of scrolling I believe, because I was running into the case of when a user adds lots of queries. I think I like this approach better (having it on `<Grid container>`)